### ISSUE TITLE
Timeline ordering

### ISSUE THREAD

**Issue by coreymwamba** [coreymwamba/uk-jazz-rhizome#8]
Would a timeline of groups be possible/trivial?? Some of the groups have start dates.

**Comment by madwort** [comment]
Any ideas what this would look like? Have lots of ideas, not sure what's the best thing. One idea is - there are currently two classes of node (person & group), how about if we introduce a third class of node (year) which groups are linked to (and maybe years are also linked to next/prev years). It may allow the force-directed graph to naturally cluster groups by year, or maybe it would just create lots of long-distance links & be illegible?

**Comment by coreymwamba** [comment]
Maybe I am overthinking this. What if - instead of a timeline - we had a slider control with a year axis (1900 - present) that dynamically controlled the appearance of groups?

**Comment by madwort** [comment]
what happens to musicians in that case - only show the set of musicians that are connected to at least one group currently displayed? gives a weird primacy to "being in a group" that I don't like, but otherwise we need to start collecting dates for musicians too...

**Comment by coreymwamba** [comment]
Yeah, I know. I'm simply trying to think through something that I know is asked for. I see birth dates as personal and private data, so I've steered away from collecting those (the names of musicians are personal but professional). I don't think it should be a "main view"; and maybe for musicians I can add an "active from/to" field...

**Comment by coreymwamba** [comment]
...which only cites decades rather than years. I can add that to the database and fill in some reasonable estimates for that.

**Comment by madwort** [comment]
> I see birth dates as personal and private data

I agree

> an "active from/to" field which only cites decades

yeah, that could work?

**Comment by coreymwamba** [comment]
Okay. I'm shutting myself away for a few days, so I'll do some database maintenance. I'll call the fields "active_from" and "active_to".
